src/pages/store/index.vue

```diff
@@ -91,25 +91,35 @@ export default {
     const title = this.$t('store_index_page_title');
     const description = this.$t('store_books_page_description');
     const link = [{ rel: 'canonical', href: `${this.$route.path}` }];
-    this.nftBooks.forEach(nft => {
+    const classIds = this.nftBooks.reduce((acc, cur) => {
+      if (Array.isArray(cur.classId)) {
+        return [...acc, cur.classId];
+      }
+      if (cur.classId.includes(',')) {
+        const ids = cur.classId.split(',').map(id => id.trim());
```

Comment:
I can confirm that the class ID is either a string that doesn't need to be split by `','`, or an array. However, this is also acceptable.

Reply:
maybe i should just use flat